standards/core/lightpush.md

```diff
@@ -0,0 +1,88 @@
+---
+title: WAKU-LIGHTPUSH
+name: Waku Light Push
+editor: Zoltan Nagy <zoltan@status.im> 
+contributors: 
+  - Hanno Cornelius <hanno@status.im>
+  - Daniel Kaiser <danielkaiser@status.im>
+  - Oskar Thorén <oskarth@titanproxy.com>
+---
+
+previous version: `/vac/waku/lightpush/2.0.0-beta1` [19/WAKU2-LIGHTPUSH](https://rfc.vac.dev/waku/standards/core/19/lightpush)
+
+---
+**Protocol identifier**: `/vac/waku/lightpush/3.0.0`
+
+## Motivation and Goals
+
+Light nodes with short connection windows and limited bandwidth wish to push messages to other nodes in the Waku network to request message services.
+A common use case is to request that the service node publish the message to an `11/WAKU2-RELAY` pubsub-topic.
+Additionally, there is sometimes a need for confirmation that a message has been received "by the network"
+(here, at least one node).
+
+`WAKU-LIGHTPUSH` is a request/response protocol for this.
+
+## Payloads
+
+```protobuf
+syntax = "proto3";
+
+message LightPushRequest {
+    string request_id = 1;
+    uin32 request_type = 10;  // 10 Reserved for future `request_type`. Currently, RELAY is the only available service.
```

Comment:
```suggestion
    // 10 Reserved for future `request_type`. Currently, RELAY is the only available service.
```

I'd still leave this field out completely. In other words, add the comment but not the field.